app/views/admin/submissions/_rejection_reason.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class='col-md-9' id='rejected-reason'>
+  <%= @submission.rejection_reason == '' ?  'None' : @submission.rejection_reason %>
```

Comment:
(question, non-blocking) Can't we just have an empty string if there's no rejection reason?

Reply:
Yes, I think it will be even better

Reply:
👍